coderd/database/queries/activitybump.sql

```diff
@@ -63,6 +68,8 @@ SET
 FROM latest l
 WHERE wb.id = l.build_id
 AND l.job_completed_at IS NOT NULL
+-- We only bump if the template has an activity bump duration set.
+AND l.activity_bump IS NOT NULL
```

Comment:
This can never be null. Should this be `= 0` or something?

Reply:
I think this didn't trip up tests since this is essentially doing nothing in the query... if it's 0 we're adding 0 to the current deadline anyways lol.

I fixed it anyways since it's better to be safe IMO